how-to/how-to-use-rootless-kata-with-podman.md

```diff
@@ -0,0 +1,108 @@
+[slirp4netns](https://github.com/rootless-containers/slirp4netns) for user-space networking
+
+## Requirements
+- A Linux system, see [supported distributions](https://github.com/kata-containers/documentation/blob/master/install/README.md#supported-distributions) for an updated list
+  - If using Centos7 follow [this guideline](https://www.scrivano.org/2018/10/12/rootless-podman-from-upstream-on-centos-7/) to correctly setup namespaces (*This should be fixed in Rhel 7.6*).
+- Golang 1.12
+
+## Installation
+ 
+To install Podman view [Installation instructions](https://github.com/containers/libpod/blob/master/install.md)<br>
+To install Kata-Containers view the [installation guidelines](https://github.com/kata-containers/documentation/blob/master/install/README.md)
+
+The following are the minimum required versions:
+* **Podman-xx.xx.xx**
+* **slirp4netns-0.4.0**
+* **Kata-Containers-[runtime](https://github.com/kata-containers/runtime)-1.9.0**
+
+## Configuration
+Now that Kata Containers and Podman have been installed, they need to be configured for rootless execution
+
+#### 1. Disable SELinux
+If enabled, disable SELinux. Kata Containers does not support SELinux; it is a known [limitation](https://github.com/kata-containers/documentation/blob/master/Limitations.md#selinux-support)
+<br>*Note: this command may differ depending on the distro being used*
+
+```bash
+sudo sed -i 's/^SELINUX=.*/SELINUX=disabled/g' /etc/selinux/config
+```
+
+#### 2. Add user to KVM group
+If running a KVM based hypervisor, the user running the workload needs to be added to the KVM group
+```bash
+sudo usermod -a -G kvm $USER
+```
+
+#### 3. Reboot
+Reboot the system for the first two steps to take effect. (*Reboot is required for SELinux change, logging out and logging back is enough for the user change to work*)
+```bash
+# To check changes took effect
+$ getenforce
+Disabled
+
+$ groups
+username kvm
+```
+
+#### 4. Disable vhost-net
+In the Kata configuration file, disable vhost-net, by uncommenting out the `disable_vhost_net = true` line <br>
+```bash
+# if the configuration.toml file already exists in /etc and is owned by user, skip these two lines
+sudo mkdir -p /etc/kata-containers/
+sudo install -o ${USER} -g root -m 0640 /usr/share/defaults/kata-containers/configuration.toml /etc/kata-containers
+
+sudo sed -i -e 's/^#disable_vhost_net = true/disable_vhost_net = true/' /etc/kata-containers/configuration.toml
+```
+
+#### 5. Modify Kata image permissions
+The Kata images needs to be accessible by the user, so change the permissions of the image directory to be owned by the user, instead of root.
+```bash
+sudo chown -R ${USER}:root /usr/share/kata-containers
+```
+#### 6. Set up Podman rootless configuration
+If `libpod.conf` does not exist in `~/.config/containers/`:
+```bash
+mkdir -p ~/.config/containers
+sudo install -o ${USER} -g ${USER} -m 0640 /usr/share/containers/libpod.conf ~/.config/containers/
```

Comment:
And *iff* we really need to do this setup, do we really need that `sudo` call?

Reply:
@marcov when you say "first startup" does it require an initial run first that will fail, and then it will exist? (That *think* that is what I was experiencing that initiated this line, but I'm not sure)

@jodh-intel, good call! I will double check that if we figure this line is necessary :)

Reply:
@gabibeyer: with podman 1.5.1 the full `~/.config/containers` folder, if not existing, is populated transparently, whatever command is executed and without failing.
I confirm that `tmp_dir` is set to `tmp_dir = "/run/user/1000/libpod/tmp"`
And the generated `libpod.conf` reuses in part what's set in `/etc/containers/libpod.conf`.

So, if the generated file is good enough, we can probably omit this section.

Reply:
@marcov What if a user wants to add the kata-runtime to the config file (following and it doesn't exist yet - their first run. Should I add that as a note in step 8? Something like 'if `libpod.conf` does not exist, run podman once using the fully qualified path and the configuration file will be populated'

Reply:
You can make that explicit, saying e.g.:
````
Verify that podman is correctly installed with:
```
$ podman info
```

This will also automatically generate a default configuration file in `~/.config/containers/libpod.conf`.
 
Add now `kata-runtime` to this files .... with:

````